src/tools/singlejar/test_util.cc

```diff
@@ -114,4 +135,19 @@ string CreateTextFile(const string& relpath, const char *contents) {
   return string("");
 }
 
+Runfiles* runfiles = nullptr;
+
 }  // namespace singlejar_test_util
+
+int main(int argc, char** argv) {
+  std::string error;
+  std::unique_ptr<singlejar_test_util::Runfiles> runfiles(
+      singlejar_test_util::Runfiles::Create(argv[0], &error));
```

Comment:
Unfortunately I can't fix the error without patching the runfiles library, which I'm about to do. Could you please hold this PR until that one is merged?

My patch will, among other things, add a static `Runfiles* Runfiles::CreateForTest(void)` method, which won't require `argv0`. Therefore there'll be no need for a custom `main` method.

Once my patch is in, could you please init the `runfiles` either on-demand (in tests that need it) or in the `setUp` method, using the upcoming `CreateForTest` factory method?

Reply:
> Could you please hold this PR until that one is merged?

Sure thing. Ping me when that PR is merged. #6251 probably will need some changes too, so don't review that one yet.